Sources/AltSwiftUI/Source/Views/ReadOnly/Text.swift

```diff
@@ -174,6 +174,7 @@ extension Text: Renderable {
             label.textColor = accentColor
         }
         label.font = context.viewValues?.font?.font
+        label.adjustsFontForContentSizeCategory = true
```

Comment:
since you are always setting true you can move this code to `createView` instead

Reply:
Thanks. I have just put this statement into `createView`